### PR TITLE
Update Community and Support links

### PR DESCRIPTION
**Pull request by shamilkhan**

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -4,7 +4,7 @@ HTTPS=true
 REACT_APP_ENV=development
 
 REACT_APP_SUPPORT_SERVICE="Telegram"
-REACT_APP_SUPPORT_URL="https://t.me/ceretroubleshooting"
+REACT_APP_SUPPORT_URL="http://cere.network/discord"
```
Comment (AndreiNavoichyk) [inline .env.dev]: `https`?

File: .env.prod [hunk 2]
```diff
@@ -4,7 +4,7 @@ HTTPS=true
 REACT_APP_ENV=production
 
 REACT_APP_SUPPORT_SERVICE="Telegram"
-REACT_APP_SUPPORT_URL="https://t.me/ceretroubleshooting"
+REACT_APP_SUPPORT_URL="http://cere.network/discord"
```
Comment (AndreiNavoichyk) [inline .env.prod]: `https`?

File: .env.dev (hunk 1 above)
Comment (shamilkhan) [reply .env.dev]: Fixed.

File: .env.prod (hunk 2 above)
Comment (shamilkhan) [reply .env.prod]: Fixed.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**